src/kaleidoscope/plugin/LED-AlphaSquare/Effect.cpp

```diff
@@ -23,7 +23,7 @@ namespace plugin {
 
 uint16_t AlphaSquareEffect::length = 1000;
 
-AlphaSquareEffect::TransientLEDMode::TransientLEDMode(AlphaSquareEffect */*parent*/)
+AlphaSquareEffect::TransientLEDMode::TransientLEDMode(AlphaSquareEffect */*parent*/) // NOLINT(readability/casting)
```

Comment:
cpplint would complain about C-style cating here, urging us to use C++-style `static_cast<>` instead. Considering this is a function argument, and unused at that, there's no point in any casting. Ignore the warning instead.